scripts/_common.sh

```diff
@@ -19,10 +19,12 @@ exec_occ() {
       php occ --no-interaction --no-ansi "$@")
 }
 
-# Create the external storage for the home folders and enable sharing
-create_home_external_storage() {
+# Create the external storage for the given folders and enable sharing
+create_external_storage() {
+  local datadir="$1"
+  local mount_name="$2"
   local mount_id=`exec_occ files_external:create --output=json \
-      'Home' 'local' 'null::null' -c 'datadir=/home/$user' || true`
+      "$2" 'local' 'null::null' -c "datadir=$datadir" || true`
```

Comment:
Why is it written `|| true`
Could you add a comment to explain that.

Reply:
This part of the code isn't from me, and there's no comments on the original commit.
I guess that sometime this command would fails.